extensions/amp-list/amp-list.md

```diff
@@ -213,7 +213,13 @@ The `<amp-list>` element exposes a `refresh` action that other elements can refe
 
 ### Dynamic resizing
 
+[filter formats="websites, stories"]
 In several cases, we may need the `<amp-list>` to resize on user interaction. For example, when the `<amp-list>` contains an amp-accordion that a user may tap on, when the contents of the `<amp-list>` change size due to bound CSS classes, or when the number of items inside an `<amp-list>` changes due to a bound `[src]` attribute. The `changeToLayoutContainer` action handles this by changing the amp list to `layout="CONTAINER"` when triggering this action. See the following example:
+[/filter]<!-- formats="websites, stories" -->
+
+[filter formats="email"]
+In several cases, we may need the `<amp-list>` to resize on user interaction. For example, when the `<amp-list>` contains an amp-accordion that a user may tap on, when the contents of the `<amp-list>` change size due to bound CSS classes. The `changeToLayoutContainer` action handles this by changing the amp list to `layout="CONTAINER"` when triggering this action. See the following example:
+[/filter]<!-- formats="email" -->
 
 ```html
 <button on="list.changeToLayoutContainer()">Show Grid</button>
```

Comment:
Just noticed, this should be: `<button on="tap:list.changeToLayoutContainer()">Show Grid</button>`